package.json

```diff
@@ -22,6 +22,7 @@
     "moment": "^2.22.0",
     "prop-types": "^15.6.1",
     "react": "^16.2.0",
+    "react-autosuggest": "^9.3.4",
```

Comment:
Is this used anywhere?

Reply:
Good catch. I started with react-autosuggest, but ended up rolling my own. I will remove it.